adapter/pkg/soaputils/soap.go

```diff
@@ -0,0 +1,135 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package soaputils
+
+import (
+	"encoding/xml"
+	"fmt"
+
+	logger "github.com/wso2/product-microgateway/adapter/pkg/loggers"
+	"github.com/wso2/product-microgateway/adapter/pkg/logging"
+)
+
+const (
+	// xmlHeader Header A generic XML header suitable for use with the output of Marshal.\
```

Comment:
```suggestion
	// xmlHeader Header A generic XML header suitable for use with the output of Marshal.
```

Reply:
fixed in https://github.com/wso2/product-microgateway/pull/2972/commits/5c91048f9fa451e4b2f2d91dac704d3f6b755cd3